src/solver_service_node.cpp

```diff
@@ -5,7 +5,7 @@
 
 extern "C"
 {
-    SOLVER_EXT_FUNC ext_func = &JackalFORCESNLPsolver_adtool2forces;
+    SOLVER_EXT_FUNC ext_func = &jackal_n3_01_H10_noSlack_adtool2forces;
```

Comment:
A bit weird that you still have the jackal version here, but my_solver in the other places (maybe it is work-in-progress ;) )